src/components/IconLibrary/iconMetaData.js

```diff
@@ -0,0 +1,28 @@
+import { keyBy } from 'lodash';
+import { icons as iconMetaData } from '@carbon/icons/metadata.json';
+import { pascal } from 'change-case';
+
+const flattenedIconMetaData = iconMetaData.flatMap(
```

Comment:
You had said before that it'd be super helpful for us to expose a flattened list of icons right? Similar to what's in build-info? Trying to figure out what I can fit into the icon-cli stuff so that we can offer this stuff for anything doing icon work 🤔

Reply:
Yeah the flattened icon list and the prefix/module name would have been incredibly helpful in the implementation here